### ISSUE TITLE
quit not exit

### ISSUE THREAD

**Issue by roualdes** [bob-carpenter/stan-model-server#1]
The REPL document (doc/REPL.md), suggests the command `exit` exits the program, but in fact it is the command `quit`.

```
~/stan-model-server (main)% stan/multi/multi -s 123 -d stan/multi/multi.data.json 
exit
UNKNOWN
Unknown instruction: exit
quit
REPL quit.
~/stan-model-server (main)% 
```

Don't yet know if this should be changed in the docs alone or if something more should happen in the source code.

**Comment by bob-carpenter** [comment]
Thanks.  That's just a doc problem.   This will all change when we make the command names all the same length.  We probably want to pad with something other than space so that it's easier to display.  Like `quit------` if we make them all 10 characters long.

**Comment by bob-carpenter** [comment]
fixed with #6
